apps/prairielearn/src/tests/mocha-env.mjs

```diff
@@ -3,9 +3,6 @@ import { fileURLToPath } from 'url';
 
 const __dirname = path.dirname(fileURLToPath(import.meta.url));
 
-// Don't perform typechecking, just transpile to keep tests fast.
-process.env.TS_NODE_TRANSPILE_ONLY = 'true';
-
 // Always use the tsconfig.json file in the `src` directory, not the one in the
-// application root. `ts-node` doesn't seem to like our usage of project references.
+// application root. `ts-node` doesn't seem to understand our usage of project references.
```

Comment:
This is the root issue here. I found this issue that I think corresponds to what we're facing? https://github.com/TypeStrong/ts-node/issues/1057

Reply:
I *think* I understand what's going on here, but jeez, we are deep in the weeds 😅

Reply:
Yeah, it's super gross. LMK if you have questions, I'm happy to share context! I'm equally fine with us just moving on and acknowledging that we'll probably never touch this again.

Reply:
Yeah, I'm happy to know that you know 😅